app/templates/components/new-myreport.hbs

```diff
@@ -28,7 +28,7 @@
     <p>
       {{t 'dashboard.associatedWith'}}
       <select onchange={{action "changePrepositionalObject" value="target.value"}}>
-        <option value=null selected={{is-equal null currentPrepositionalObject}}>
+        <option value='' selected={{is-equal null currentPrepositionalObject}}>
```

Comment:
I feel weird about this change - mainly because of the comparison to `null` in the selected attribute.  Is this part of the fix?